* Fix tons of "warning: the frame size of ... bytes is larger than 0 bytes"

Index: linux-2.6.26-source/Makefile
===================================================================
--- linux-2.6.26-source.orig/Makefile
+++ linux-2.6.26-source/Makefile
@@ -507,7 +507,7 @@
 KBUILD_CFLAGS	+= -O2
 endif
 
-ifneq (CONFIG_FRAME_WARN,0)
+ifneq ($(CONFIG_FRAME_WARN),0)
 KBUILD_CFLAGS += $(call cc-option,-Wframe-larger-than=${CONFIG_FRAME_WARN})
 endif
 
